framework/wazuh/core/config/models/management_api.py

```diff
@@ -0,0 +1,102 @@
+from pydantic import BaseModel, PositiveInt
+from typing import Literal, List
+
+
+from wazuh.core.config.models.ssl_config import APISSLConfig
+from wazuh.core.config.models.logging import LoggingWithRotationConfig
+
+
+class ManagementAPIIntervals(BaseModel):
+    """Configuration for management API intervals.
+
+    Parameters
+    ----------
+    request_timeout : int
+        The timeout for requests in seconds. Default is 10.
+    """
+    request_timeout: int = 10
+
+
+class CorsConfig(BaseModel):
+    """Configuration for Cross-Origin Resource Sharing (CORS).
+
+    Parameters
+    ----------
+    enabled : bool
+        Whether CORS is enabled. Default is False.
+    source_route : str
+        The source route for CORS requests. Default is "*".
+    expose_headers : str
+        Headers that are exposed to the client. Default is "*".
+    allow_headers : str
+        Headers that are allowed in requests. Default is "*".
+    allow_credentials : bool
+        Whether to allow credentials in CORS requests. Default is False.
+    """
+    enabled: bool = False
+    source_route: str = "*"
+    expose_headers: str = "*"
+    allow_headers: str = "*"
+    allow_credentials: bool = False
+
+
+class AccessConfig(BaseModel):
+    """Configuration for access control settings.
+
+    Parameters
+    ----------
+    max_login_attempts : PositiveInt
+        The maximum number of failed login attempts allowed. Default is 50.
+    block_time : PositiveInt
+        The duration in seconds to block an IP after reaching the maximum login attempts. Default is 300.
+    max_request_per_minute : PositiveInt
+        The maximum number of requests allowed per minute. Default is 300.
+    """
+    max_login_attempts: PositiveInt = 50
+    block_time: PositiveInt = 300
+    max_request_per_minute: PositiveInt = 300
+
+
+class ManagementAPIConfig(BaseModel):
+    """Configuration for the management API.
```

Comment:
```suggestion
    """Configuration for the Management API.
```

Reply:
Changed in e4e54327c2b3a45cbe77a6f9c30cfd19dc9ce0b6